custom-words.txt

```diff
@@ -2070,29 +2080,3 @@ ziplist
 Zoho
 zoneinfo
 zset
-tpgs
```

Comment:
I don't see these words in this file any more. Why do we remove them?
```
tpgs
ProviderHub
regionality
Resource
metdata
Exprired
saskey
vmsize
FSLogix

```

Reply:
Some were unfixed typos moved to file level suppression in `cspell.json`, duplicates of words that were already in the file in the right sort location, CamelCase words, that cSpell already treats as separate words, or correctly spelled words like "Resource" that shouldn't be in this overrides dictionary